src/HTMLImportTranscoder.js

```diff
@@ -0,0 +1,99 @@
+// Copyright 2011 Traceur Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Applies Traceur to HTML Import scripts related to a Web page.
+import {WebPageTranscoder} from './WebPageTranscoder.js';
+
+var importSelector = 'link[rel=import]';
+
+export class HTMLImportTranscoder {
+
+  findAllChildrenHTMLImports_(parentImportNodes) {
+    var foundImportNodes = [];
+
+    // process any import children
+    Array.prototype.forEach.call(parentImportNodes, (link) => {
```

Comment:
Can we use c style for loops instead (or for-of loops but these are probably NodeLists)